frontend/src/pages/chat/components/chat-sidebar/styles.module.scss

```
@@ -75,3 +75,18 @@
 .plus {
   width: 34px;
 }
+
+@media (width <= 480px) {
```

please use one media style
```suggestion
@media (width < 480px) {
```

---

done